### PR TITLE
nord-theme-add

### PR DESCRIPTION
**Pull request by yavko**
[NORD](https://www.nordtheme.com/) SUPREMACY!

1. Look Sexy
2. amazing
3. lots of support
4. yavko likes it so should you
5. hahaha yavko made hehe

> made by yavko

### CODE REVIEW

File: themes/nord.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Yavko",
```
Comment (fwesss) [inline themes/nord.json]:
```suggestion
        "name": "Arctic Ice Studio & Sven Greb",
```

Attribution should go to the original theme creator. In this case, it's Arctic Ice Studio & Sven Greb.

File: themes/nord.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        "twitter": "@lolyavko",
```
Comment (fwesss) [inline themes/nord.json]:
```suggestion
        "twitter": "@arcticicestudio",
```

File: themes/nord.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "github": "yavko"
```
Comment (fwesss) [inline themes/nord.json]:
```suggestion
        "github": "arcticicestudio"
```

**Review by fwesss — CHANGES_REQUESTED** [review]
Love Nord and use it for everything! A matching Fig theme was the last thing I needed to get my whole system to match so this PR is greatly appreciated.

Only thing is we need the attribution to go to the original theme creator. In this case it's Arctic Ice Studio & Sven Greb.

File: themes/nord.json (hunk 1 above)
Comment (yavko) [reply themes/nord.json]: oops didnt know

**Pull request merged**